web/src/main/java/se/inera/intyg/rehabstod/service/sjukfall/GetActiveSickLeavesResponseServiceImpl.java

```diff
@@ -88,27 +87,34 @@ public GetActiveSickLeavesResponseDTO get(SickLeavesFilterRequestDTO filterReque
             filterRequest.getUnansweredCommunicationFilterTypeId()
         );
         final var hasDecoratedWithSRSInfo = decorateWithSRSInfo(filteredSickLeaves);
-        pdlLogSickLeavesService.log(filteredSickLeaves, ActivityType.READ, ResourceType.RESOURCE_TYPE_SJUKFALL);
-        performMonitorLogging(sickLeaves, getUnitForLogging(request));
+        performPdlLogging(filteredSickLeaves, shouldPdlLog);
```

Comment:
The code reads a bit strange :-) 

Consider renaming the method or add the if-statement directly in this code.